### PR TITLE
Feature/sensor placement hypervisor

### PR DESCRIPTION
**Pull request by mgruhler**
Just easier to discuss with a PR ;-)

### CODE REVIEW

File: seneka_sensor_placement/include/seneka_sensor_placement/sensor_placement_hypervisor.h [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (mgruhler) [inline seneka_sensor_placement/include/seneka_sensor_placement/sensor_placement_hypervisor.h]: please remove the whitespace

File: seneka_sensor_placement/src/sensor_placement_hypervisor.cpp [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    maneuver_msg.payload = maneuver_msg.PAYLOAD_PICKUP;
+
+    // publish the quanjo_maneuver message
+    quanjo_maneuver_pub_.publish(maneuver_msg);
+    ROS_INFO("quanjo_maneuver message sent with PAYLOAD_PICKUP");
+  }
+
+  //clear paths std::vector after pickup is completed TODO: check if this is the intended behaviour
+  paths_vec_.clear();
+}
```
Comment (mgruhler) [inline seneka_sensor_placement/src/sensor_placement_hypervisor.cpp:108]: This is fine like this. We assume, that after the Pickup msg, the nodes are actually picked up.
(For real scenarios, we would definitly need more intelligence here, e.g. only pick up a specified node --> nodes need IDs, or something, the nodes publish their position, take those for retrieval, etc... But we won't do this currently)

File: seneka_sensor_placement/src/sensor_placement_hypervisor.cpp [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  {
+    // create quanjo_maneuver msg for pickup
+    maneuver_msg.path = paths_vec_.at(i);
+    maneuver_msg.payload = maneuver_msg.PAYLOAD_PICKUP;
+
+    // publish the quanjo_maneuver message
+    quanjo_maneuver_pub_.publish(maneuver_msg);
+    ROS_INFO("quanjo_maneuver message sent with PAYLOAD_PICKUP");
+  }
+
```
Comment (mgruhler) [inline seneka_sensor_placement/src/sensor_placement_hypervisor.cpp:106]: The "Pickup Paths" should not be published as single paths (i.e. multiple msgs), but as a single one. So you should concatenate the different paths into a single one. Best do this direcly in the nav_path_sub_ callback...

**Comment by ipa-mig-mc** [comment]
@ipa-mig Clean up done! you can merge it now :)

**Pull request merged**